hokusai/production.yml

```diff
@@ -103,6 +106,119 @@ spec:
     layer: application
     component: web
   type: ClusterIP
+
+### for novo ###
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: force-web-novo
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: force-novo
+        layer: application
+        component: web
+      name: force-web-novo
```

Comment:
Should maybe be `...-novo-web` so the prefixes are all consistently `force-novo-`.

Reply:
Switching novo/web around should be fine. I'll update.